tests/data/constants.py

```diff
@@ -6,6 +6,12 @@
 
 ROOT_DIR = os.path.abspath(os.path.join(__file__, "../../.."))
 
+# CORD-19 paths
+metadata_path = os.path.join(ROOT_DIR, 'input/cord19/metadata.csv')
```

Comment:
I recommend adding a sample of the metadata file for testing purposes which can live in `tests/data/resources`. Maybe the first 50 lines. Adding that path in `constants.py` and then using that sample data to create a test is what I did for other data imports

Reply:
This is done now.

Reply:
Done now